src/detect-nfs-procedure.c

```diff
@@ -146,7 +146,13 @@ static int DetectNfsProcedureMatch (DetectEngineThreadCtx *det_ctx,
  */
 static DetectU32Data *DetectNfsProcedureParse(const char *rawstr)
 {
-    return DetectU32Parse(rawstr);
+    DetectU32Data *r = DetectU32Parse(rawstr);
+    // this range used to be inclusive
```

Comment:
should this be an argument to `DetectU32Parse`? e.g. `DetectU32Parse(rawstr, DETECT_U32_INCLUSIVE)`

Reply:
I wonder...

Do we prefer 
- consistency in version 7 for the use of ranges (meaning '<>' is always an exclusive range in a signature)
- consistency for a ruleset when upgrading from 6 to 7 (meaning no need to change the rules when upgrading)

Reply:
I think this is too subtle a change to impose on users. It's more something to change if we create a new version of a keyword under a new name.

Reply:
Ok, wonder if we should document the inconsistency for these nfs ranges